src/styles/blocks/variables.scss

```diff
@@ -0,0 +1,11 @@
+$square-side: 100px;
```

Comment:
It's not clear what is `square` in this task
Probably will be better to specify the name relatively to the `date` naming